### PR TITLE
Add information about which component can't pick up kubevirt config

### PR DESCRIPTION
**Pull request by iholder-redhat**
Tiny addition to include the label which has trouble picking up kubevirt updated configuration.
This should help the effort to take test 2666 out of quarantine (see PR https://github.com/kubevirt/kubevirt/pull/5669).

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -4590,7 +4590,7 @@ func WaitForConfigToBePropagatedToComponent(podLabel string, resourceVersion str
 			return compareResourceVersions(resourceVersion, result["config-resource-version"].(string))
 		}
 		return true
-	}, 10*time.Second, 1*time.Second).Should(BeTrue(), "Not all kubevirt components picked up the kubevirt config successfully")
+	}, 10*time.Second, 1*time.Second).Should(BeTrue(), "Not all kubevirt components picked up the kubevirt config successfully for pod label "+podLabel)
```
Comment (xpivarc) [inline tests/utils.go:4593]: I think we could change the signature of the function to return an error. This error could include more context(for example Pod`s name).

WDYT?

Comment (iholder-redhat) [reply tests/utils.go:4593]: Sounds good!

Comment (enp0s3) [reply tests/utils.go]: @xpivarc @iholder-redhat I think this is a good idea, but it think it deserves a separate PR of refactoring context. Let's take this change so we will be able to find the root cause of the CI issues sooner.

Comment (enp0s3) [reply tests/utils.go]: @xpivarc @iholder-redhat I see that code was already updated. Please ignore my previous comment. I just didn't see until now that the `Eventually` can return an error and treated by `ShouldNot(HaveOccured())

**Review by enp0s3 — APPROVED** [review]
/approve

**Comment by xpivarc** [comment]
/lgtm
Let`s see if we can uncover the problem in CI.

**Comment by rmohr** [comment]
@iholder-redhat the iformation would have been in the logs which we collect with the docs. Why did you not look there? Now it is not in the logs anymore. Can you revert this?

**Comment by rmohr** [comment]
> @iholder-redhat the iformation would have been in the build logs. Why did you not look there? Now it is not in the logs anymore. Can you revert this?

Never mind. Actually in the last case we were not printing the pod name.

File: tests/utils.go [hunk 2]
```diff
@@ -4564,33 +4564,37 @@ func waitForConfigToBePropagated(resourceVersion string) {
...
+	EventuallyWithOffset(3, func() error {
 		pods, err := virtClient.CoreV1().Pods(flags.KubeVirtInstallNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: podLabel})
+
 		if err != nil {
-			log.DefaultLogger().Reason(err).Infof("Failed to fetch pods.")
-			return false
+			return fmt.Errorf("failed to fetch pods. %s", errAdditionalInfo)
 		}
 		for _, pod := range pods.Items {
+			errAdditionalInfo += fmt.Sprintf(", pod: \"%s\"", pod.Name)
```
Comment (rmohr) [inline tests/utils.go:4576]: if I read the code correct this will add every pod name to errAdditionalInfo accumulating them.

Comment (iholder-redhat) [reply tests/utils.go:4576]: Thanks!
PR: https://github.com/kubevirt/kubevirt/pull/5776

**Pull request merged**